.circleci/config.yml

```diff
@@ -278,16 +278,16 @@ jobs:
             npm --version
       - when:
           condition:
-            not: << pipeline.parameters.sanity-check-against-develop >>      
-          steps:  
+            not: << pipeline.parameters.sanity-check-against-develop >>
+          steps:
             - get_workspace
             # Override the webservice version in package.json to instead build against develop webservice
             - run:
                 name: set webservice to develop
                 command: bash -i -c 'npm config set dockstore-ui2:webservice_version develop'
```

Comment:
Do you still need to set the version to develop if you're using the snapshot version?